File: Python/1020_IdadeEmDias.py
# Entrada de Dados
idadeDias = int(input())

# Cálculo
anos = idadeDias/365
idadeDias %= 365

meses = idadeDias/30
idadeDias %= 30

# Saída de Dados
print(f"{int(anos)} ano(s)")
print(f"{int(meses)} mes(es)")
print(f"{int(idadeDias)} dia(s)")

File: C/1020_idade_em_dias.c
#include <stdio.h>

int main() {

    int idadeDias, ano, meses;
    scanf("%d", &idadeDias);

    ano = idadeDias/365;
    idadeDias %= 365;

    meses = idadeDias/30;
    idadeDias %= 30;

    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", meses);;
    printf("%d dia(s)\n", idadeDias);

    return 0;
}